src/avfoundation.cs

```diff
@@ -8216,6 +8216,37 @@ interface AVCaptureStillImageOutput {
 #endif
 	}
 
+	[NoTV, iOS (10,0), NoMac, NoWatch]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface AVCaptureDeviceDiscoverySession {
+
+		[Internal]
+		[Static]
+		[Export ("discoverySessionWithDeviceTypes:mediaType:position:")]
+		AVCaptureDeviceDiscoverySession _Create (NSString [] deviceTypes, [NullAllowed] string mediaType, AVCaptureDevicePosition position);
+
+		[Export ("devices")]
+		AVCaptureDevice [] Devices { get; }
+	}
+
+	[NoTV, iOS (10,0), NoMac, NoWatch]
+	[Flags]
```

Comment:
did you check the generated code ?
I'm not sure I considered flags when dealing with constants...